[Fluent] Hide coinjoins in UI

soosr opened the pull request:
part of https://github.com/zkSNACKs/WalletWasabi/issues/6422

This PR hides the CJs in the history table and prevents CJ notifications. 

Note that the CJ transaction detection is not working correctly at the moment. https://github.com/zkSNACKs/WalletWasabi/issues/6205

yahiheb commented:
Tested, LGTM

MaxHillebrand reviewed (approved):
tACK 0338bdb

MaxHillebrand reviewed (changes requested):
tested 37c20266607093f9780aee6a34af00504502919b

I still get some transactions listed as CoinJoin...
![screenshot_20210927_101112](https://user-images.githubusercontent.com/30683012/134869821-38b8b536-b410-434b-b921-fb4c8bfcbf75.png)

MaxHillebrand reviewed (commented):
tested 0cf7b17


In the transaction details dialog, there is no longer a copy button at the tx id. 

---

FTR...
![screenshot_20211006_182614](https://user-images.githubusercontent.com/30683012/136244704-b0f3af37-243b-42a5-ac36-e805ad7e1562.png)

We should figure out what useful information to put here later.

soosr commented:
> the transaction id has only one copy button for many ids, and it puts them both together to the clipboard in one long string.

Honestly, I don't want to implement it first. Because it requires too much code for a sake of a copy button. I was like... If someone wants to copy only one transaction ID, then they will select it and copy it manually. Although it was a test too. If someone will notice it and complain, I will implement it.

Fixed https://github.com/zkSNACKs/WalletWasabi/pull/6424/commits/2de6633e3f15a1a96436033155f36f3aa1990a73

> it says confirme, missing the d

A bug in Avalonia. https://github.com/zkSNACKs/WalletWasabi/issues/6498

> Amount should be Mining fee paid

fixed

> In the tx details it should split up the stats per coinjoin, right now that is only the fee paid, but in the future it can be more stats like anonset, input/output count etc.

Yeah, let's gather some ideas and we can add them in a separate PR.
